README.md

```diff
@@ -1,222 +1,211 @@
+  **Correct:**
 
-Get latest changes.
+  Variables defined at top of file, re-using existing variables where sensible.
 
 ```
-git pull
-npm install
-```
-If errors or warnings (as with local installation):
-```npm audit fix```
-
-Kill existing NPM process (stop the server). 
-The process ID will vary, but as an example:
-```ps -ef | grep npm```
-Will display
-```````
-tsg       5849     1  0 May07 ?        00:00:00 npm
-tsg      23701 22062  0 12:21 pts/1    00:00:00 grep npm
+$component-color: #FF0000;
+$component-font-size: map-deep-get($header-styles, 'small', 'h4', 'font-size');
 ```
-So:
-```kill 5849```
-And it's dead.  
 
-Restarting the server
-```nohup npm start > tsg.log 2>&1 &```
-nohup ensures that the server is running in daemon mode and will not stop when the session is closed.
+**`map-deep-get`** Gets the font-size of h4 elements from Foundation settings.
 
-Checking logs
-```tail -f tsg.log````
 
-## stuff worth checking
-- smacss.com
-- bem.info
-- [Foundation docs]
-- [sc5 docs]
-- [sc5 build options]
+By wrapping in smaller mixins, we allow the end user to choose if they want to include our entire component style definitions or only part of them.
 
-## Working with the styleguide
+Possibility to pass a variable with a default value `($font-size: $base-font-size)` makes the mixin more versatile.
 
-SC5-Styleguide, parses SCSS comments for a standalone component library.
 
-Currently existing styles, should be most up-to-date but this tool will not be maintained in the future:
-https://company-81847.frontify.com/d/VHfNy7brWRvI/tieto-style-guide#/introduction/overview
+ ```
+@mixin component-font-styles($font-size: $base-font-size)
+  @debug '$font-size is #{$font-size}';
+  font-size: $font-size;
+  margin-bottom: $global-margin;
+}
+```
+
+```
+@mixin my-component {
+  .my-component {
+    @include component-font-styles($component-font-size);
+    color: $component-color;
+  }
+}
+```
+
+Don't forget that Sass has a lot of built-in features making it more like a programming language than writing CSS. See https://sass-lang.com/documentation or http://thesassway.com.
 
-SC5-styleguide
-https://github.com/SC5/sc5-styleguide
+## Logs
 
-Built on KSS documenting syntax
-http://warpspire.com/kss/
+You can check logs on DEMO environment (locally, you can check the log in the terminal where you NPM started the styleguide) by navigating into the folder of styleguide and running command **`tail -f tsg.log`**.
 
-Build options
-https://github.com/SC5/sc5-styleguide#build-options
+Alternatively, you can check the whole log file through text editor of your choice (nano for example) or **`cat`** command.
 
-Check links in the Additional Info section of the github project. There are some useful things for advanced techniques.
+## Template for standalone
 
-### Template for standalone
+@TODO - no idea what was meant to be here, Liisa.
```

Comment:
I need to research how that template thing works, I don't recall anymore. So we can make our own custom markup for this.